fission/src/systems/MatchMode.ts

```diff
@@ -17,13 +18,24 @@ class MatchMode {
     private timeLeft: number = 0
     private intervalId: number | null = null
 
+    // Match Mode Config
+    private autonomousTime: number = 15
+    private teleopTime: number = 135
+    private endgameTime: number = 20
+
     private constructor() {}
 
     static getInstance(): MatchMode {
         MatchMode.instance ??= new MatchMode()
         return MatchMode.instance
     }
 
+    setMatchModeConfig(config: MatchModeConfig) {
+        this.autonomousTime = config.autonomousTime
+        this.teleopTime = config.teleopTime
+        this.endgameTime = config.endgameTime
+    }
```

Comment:
I feel like it makes more sense to have there be a `MatchModeConfig` field on `MatchMode` rather than transferring individual properties. That way it's simpler to extend when more properties are added